cmd/system-probe/config/adjust_npm.go

```diff
@@ -24,6 +24,8 @@ const (
 )
 
 func adjustNetwork(cfg config.Config) {
+	ebpflessEnabled := cfg.GetBool(netNS("enable_ebpf_less"))
```

Comment:
nit suggestion: rename to `enable_ebpfless`

Reply:
Done